### PR TITLE
Disallow uint8 indices.

### PR DESCRIPTION
**Pull request by donmccurdy**
/cc https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/329

### CODE REVIEW

**Comment by UX3D-nopper** [comment]
see my comment https://github.com/KhronosGroup/glTF/issues/1471

We cannot just remove it, as some people maybe need it.

**Comment by lexaknyazev** [comment]
For compatibility, can we keep uint8 import while removing uint8 export?

**Comment by lexaknyazev** [comment]
Also, regardless of the related spec issue outcome, `65536` should be changed to `65535` to avoid primitive restart.

**Comment by emackey** [comment]
> We cannot just remove it, as some people maybe need it.

Are there people who specifically need UINT8 export from Blender via this plugin?  That seems like too small an edge case.

FWIW, in my conversations with artists, the "maximum indicies" options with different integer sizes are the most confusing of all this plugin's options.  I think it's the only one that doesn't reference a feature they know about from Blender, but rather references an internal detail of the file storage format that they've never heard of.

**Comment by donmccurdy** [comment]
Thanks! I've reduced the maximum index values by 1, per https://github.com/KhronosGroup/glTF/pull/1476/files.

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_generate.py [hunk 1]
```diff
@@ -1132,15 +1132,13 @@ def generate_meshes(operator,
...
-            if max_index < 256:
-                componentType = GLTF_COMPONENT_TYPE_UNSIGNED_BYTE
-            elif max_index < 65536:
+            if max_index < 65535:
                 componentType = GLTF_COMPONENT_TYPE_UNSIGNED_SHORT
-            elif max_index < 4294967296:
+            elif max_index < 4294967295:
                 componentType = GLTF_COMPONENT_TYPE_UNSIGNED_INT
             else:
                 print_console('ERROR', 'Invalid max_index: ' + str(max_index))
```
Comment (lexaknyazev) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_generate.py]: Could the error message here be more descriptive? Something like:
> A mesh contains too many vertices and needs to be split before export.

Comment (donmccurdy) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_generate.py]: Done

**Pull request merged**